### PR TITLE
(DOCSP-8551) Add example queries to Query Your Data page

### PR DESCRIPTION
**Pull request by zach-carr**
[Ticket](https://jira.mongodb.org/browse/DOCSP-8551)
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/compass/zach.carr/DOCSP-8551/query/filter.html#examples)

- Added the Examples section to the Query Your Data page
- Added sample data (two documents) and procedure for inserting through Compass
- Linked to Atlas sample data
- Added 4 examples: match, condition, date and array queries
- Linked to manual for more info/ examples

### CODE REVIEW

File: source/query/filter.txt [hunk 1]
```diff
@@ -108,6 +108,172 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
...
+      }
+
+Filter by Elements in an Array
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where ``scores`` 
+   contains at least one number greater than ``80`` and less than 
+   ``90`` using the
+   :manual:`$elemMatch operator </reference/operator/query/elemMatch/>`.
```
Comment (kanchana-mongodb) [inline source/query/filter.txt]: Consider rephrasing to something similar to the following:

The following $filter query uses the :manual:`$elemMatch operator </reference/operator/query/elemMatch/>` to find documents where at least one value in the ``scores`` array is greater than ``80`` and less than 90.

File: source/query/filter.txt [hunk 2]
```diff
@@ -108,6 +108,172 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
...
+   If you do not have a MonogDB deployment or if you would like to 
+   query a complete sample data set, see
+   :atlas:`Sample Data for Atlas Clusters</sample-data/>` for 
+   instructions on creating a free-tier cluster with sample data.
+
+Filter for a Match
+~~~~~~~~~~~~~~~~~~
+.. example::
+   
+   The following filter query finds all documents where
```
[the remark this answers is not in the document]
Comment (zach-carr) [reply source/query/filter.txt]: It should probably be "query filter" and link to the top of the page: https://docs.mongodb.com/compass/master/query/filter/#set-query-filter

I'll change that in all the examples

File: source/query/filter.txt (hunk 1 above)
Comment (zach-carr) [reply source/query/filter.txt]: Rephrasing all examples this way

File: source/query/filter.txt [hunk 3]
```diff
@@ -108,6 +108,172 @@ How Does the Compass Query Compare to MongoDB and SQL Queries?
          SELECT * FROM article
          WHERE author = "Joe Bloggs";
 
+Examples
+--------
+
+.. class:: example
+
+   You can use the following two JSON documents as sample data. To 
+   import the data to your MongoDB deployment with |compass|:
```
[the remark this answers is not in the document]
Comment (zach-carr) [reply source/query/filter.txt]: Rephrased to: 

"The following examples use the JSON documents below as sample data. To import this sample data to your MongoDB deployment with |compass|:"

**Comment by zach-carr** [comment]
Good point, probably not - I'll update the examples to use only the date without a time.

**Comment by zach-carr** [comment]
Good point, I think it's worth changing.

- "age" to "version"
- $gte example now $lte to diversify operators
- tabbed out examples and removed example block styling per Jeff's recommendation

**Pull request merged**